### PR TITLE
Add bcoroneliit to CODEOWNERS

### PR DESCRIPTION
**Pull request by ReneOv-MSFT**

### CODE REVIEW

**Review by gingi — APPROVED** [review]

**Review by skapur12 — CHANGES_REQUESTED** [review]
BTW, just thought I'd mention that Dave mentioned to me that we weren't going to add interns to codeowners because you only need to be in codeowners to approve PRs and we can approve their PRs- but let me know if that's not the case

**Pull request closed without merging**
